src/azure-cli/azure/cli/command_modules/profile/custom.py

```diff
@@ -74,6 +74,15 @@ def get_access_token(cmd, subscription=None, resource=None, resource_type=None,
     profile = Profile(cli_ctx=cmd.cli_ctx)
     creds, subscription, tenant = profile.get_raw_token(subscription=subscription, resource=resource, tenant=tenant)
 
+    token_entry = creds[2]
+    # MSIAuthentication's token entry has `expires_on`, while ADAL's token entry has `expiresOn`
+    # Unify to UTC `expiresOn`, like "2020-06-30 06:14:41"
+    if 'expires_on' in token_entry:
+        from datetime import datetime
+        # https://docs.python.org/3.8/library/datetime.html#strftime-and-strptime-format-codes
+        token_entry['expiresOn'] = datetime.fromtimestamp(int(token_entry['expires_on']))\
+            .strftime("%Y-%m-%d %H:%M:%S.%f")
```

Comment:
can we use local time instead of UTC time ?

Reply:
This is local time. For UTC time, we use `utcfromtimestamp` instead.